src/modules/manaFiatGateway/selectors.ts

```diff
@@ -8,9 +9,15 @@ import { ManaFiatGatewayState } from './reducer'
 export const getState: (state: any) => ManaFiatGatewayState = state =>
   state.manaFiatGateway
 
+export const getData = (state: any) => getState(state).data
 export const getLoading = (state: any) => getState(state).loading
 export const getError = (state: any) => getState(state).error
 
+export const getPendingPurchase = (state: any) =>
```

Comment:
```suggestion
export const getPendingPurchase = (state: ManaFiatGatewayState) =>
```
I know the other ones were already there and they all have `any`, but I believe we could start typing it as `ManaFiatGatewayState`, right?

Reply:
I think it should be `RootState` instead of the  `ManaFiatGatewayState`. The second type is what you get after getting the `manaFiatGateway` from the `state: RootState`.

Reply:
Oh yeah, `RootState`, sorry, that's right!